src/app/static/src/app/components/genericChart/GenericChart.vue

```diff
@@ -60,7 +60,7 @@
                 </div>
                 <div v-for="dataSource in chartConfigValues.dataSourceConfigValues.filter(ds => ds.tableConfig)"
                      :key="dataSource.config.id">
-                    <download-indicator :filtered-data="filteredDataWithoutPages![dataSource.config.id]"
+                    <download-indicator :filtered-data="filteredDataWithoutPages ? filteredDataWithoutPages[dataSource.config.id] : []"
```

Comment:
I thought it was safe to assume `filteredDataWithoutPages` would be truthy, so you could remove nullable option from the Computed interface?

Reply:
unfortunately the computed value actually explicitly returns a null in one of the cases, I tried replacing this with a `{}` but this caused errors, so I just put a ternary